vignettes/archive.Rmd

```diff
@@ -117,6 +147,9 @@ object:
 * `geo_type`: the type for the geo values.
 * `time_type`: the type for the time values.
 * `additional_metadata`: list of additional metadata for the data archive.
+* `compactify`: TRUE eliminates redundant entries using LOCF for efficiency;
+no entry also brings up a message to indicate which columns should be removed
+for efficiency; and FALSE does not eliminate redundant entries.
```

Comment:
I don't think `compactify` is actually part of the metadata:
```
# `dt` here is taken from the tests
as_epi_archive(dt,compactify=TRUE)$geo_type # "state"
as_epi_archive(dt,compactify=TRUE)$compactify # NULL
```

Two approaches:
- Just remove this line.
- Try to make most of this line true.  Just adding `compactify` as a field would be a bit confusing; I'd expect `my_archive$compactify` to be a function that would perform compactification (--- that might be an idea to implement later).  So instead the metadata field would probably need to be called something like `compactified`.

Reply:
I updated this on the documentation.